### DISCUSSION TITLE
git branch -D

### DISCUSSION THREAD

**Discussion by minhDtran** [community/community#23255] [New to GitHub]
<p>Hello,</p>
<p>   I am new to Git , and I would like to learn Git command.</p>
<p>I would like to know what does git branch -D mean ? I assume that git branch create a branch in git; for example, git branch test creating a test branch. But I do  not know what does the parameter -D mean in git branch -D . Could someone please help ? Thanks alot.</p>
<p>Also what does git tags -D mean ? I assume that git tags test meaning assign name of test to the branch that I just created from teh previous command such as git branch test. But i do not know what does git tags -D mean ?</p>
<p>Thanks for all of the helps.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/minhdtran">@minhdtran</a>,</p>
<p>Thank you for being here! Taking a look at your command:</p>
<pre><code>git tags -D
</code></pre>
<p>I believe the <code>-D</code> on that command force delete the tags. Here is a handy guide about deleting branches, that might be useful: <a href="https://www.freecodecamp.org/news/how-to-delete-a-git-branch-both-locally-and-remotely/" rel="nofollow noopener">https://www.freecodecamp.org/news/how-to-delete-a-git-branch-both-locally-and-remotely/</a></p>
